src/components/transactions/StakeRewardClaim/StakeRewardClaimActions.tsx

```diff
@@ -33,7 +33,7 @@ export const StakeRewardClaimActions = ({
         amount: amountToClaim,
       });
     },
-    skip: blocked,
+    skip: !amountToClaim || parseFloat(amountToClaim) === 0 || blocked,
     deps: [],
```

Comment:
Add `amountToClaim` as a dep here so the tx is updated when the value gets changed